presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -586,16 +588,12 @@ public static int compareUnsigned(Slice left, Slice right)
 
     public static int compareUnsigned(long leftRawLo, long leftRawHi, long rightRawLo, long rightRawHi)
```

Comment:
I'd rather leave all arguments to be in the same form and unpack internally in the method, as @sopel39 suggested in another comment